packages/plugin-nm/sources/NodeModulesLinker.ts

```diff
@@ -756,14 +770,15 @@ type DirEntry = {
   kind: DirEntryKind.FILE;
   mode: number;
   digest?: string;
+  mtimeMs?: number;
 } | {
   kind: DirEntryKind. DIRECTORY;
 } | {
   kind: DirEntryKind.SYMLINK;
   symlinkTo: PortablePath;
 };
 
-const copyPromise = async (dstDir: PortablePath, srcDir: PortablePath, {baseFs, globalHardlinksStore, nmMode, packageChecksum}: {baseFs: FakeFS<PortablePath>, globalHardlinksStore: PortablePath | null, nmMode: {value: NodeModulesMode}, packageChecksum: string | null}) => {
+export const copyPromise = async (dstDir: PortablePath, srcDir: PortablePath, {baseFs, hardlinksStorePath, nmMode, packageChecksum}: {baseFs: FakeFS<PortablePath>, hardlinksStorePath: PortablePath | null, nmMode: {value: NodeModulesMode}, packageChecksum: string | null}) => {
```

Comment:
I'm really not fan of having a separate `copy` function... It was fine when it was bound to the nm linker, but if it leaks into the others it starts to become concerning imo 🤔